app/src/main/AndroidManifest.xml

```diff
@@ -27,6 +28,7 @@
             android:screenOrientation="landscape">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
+                <category android:name="android.intent.category.LAUNCHER" />
```

Comment:
Hm, this is nice, but I believe this will make the app available for phones as well or not?

Reply:
If that is the case then we will have a problem as the current layout doesn't really work for phones